examples/meta/src/clustering/hierarchical.sg

```diff
@@ -1,23 +1,17 @@
-CSVFile f_feats_train("../../data/classifier_4class_2d_linear_features_train.dat")
+File f_features_train = csv_file("../../data/classifier_4class_2d_linear_features_train.dat")
 
 #![create_features]
-RealFeatures features_train(f_feats_train)
+Features features_train = features(f_features_train)
 #![create_features]
 
 #![choose_distance]
-EuclideanDistance distance(features_train, features_train)
+Distance d = distance("EuclideanDistance", lhs=features_train, rhs=features_train)
 #![choose_distance]
 
 #![create_instance]
-int merges = 3
-Hierarchical hierarchical(merges, distance)
+Machine hierarchical = machine("Hierarchical", merges=3, distance=d)
 #![create_instance]
 
 #![train_model]
-hierarchical.train()
+hierarchical.train(features_train)
 #![train_model]
-
-#![extract_results]
-RealVector d = hierarchical.get_merge_distances()
-IntMatrix cp = hierarchical.get_cluster_pairs()
```

Comment:
These ones had to go because the members are T*. These get_* methods wrap the T* into SGVector or SGMatrix.